health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/securityPopUp.js

```
@@ -0,0 +1,93 @@
+    setAnswers((prev) => ({
+      ...prev,
+      [id]: value,
+    }));
+  };
+
+  return (
+    <PopUp
+      onClose={onClose}
+      heading={t(`HCM_MICROPLAN_VILLAGE_SECURITY_LEVEL_LABEL`)}
+      onOverlayClick={onClose}
+      children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)